C-Plus-Plus/README.md

```diff
@@ -50,6 +50,7 @@
 - [Sub-array with given sum](cp/SubArrayWithGivenSum.cpp)
 - [Suduko Solver](cp/SudukoSolver.cpp)
 - [Target Sum triplets](cp/target_sum_triplets.cpp)
+- [Toggle Kth Bit From Right](ToggleKthBitFromRight.cpp)
```

Comment:
@mansi-9 The path should be `- [Toggle Kth Bit From Right](cp/ToggleKthBitFromRight.cpp)`.

Reply:
Sure. Thanx!!!